user/helm-chart/README.md

```diff
@@ -0,0 +1,58 @@
+# Deploy **KubeStellar** service in a cluster using Helm
+
+Table of contests:
+- [Deploy **KubeStellar** service in a cluster using Helm](#deploy-kubestellar-service-in-a-cluster-using-helm)
+  - [Deploy **KubeStellar** in a **Kubernetes** cluster (**Kind** cluster)](#deploy-kubestellar-in-a-kubernetes-cluster-kind-cluster)
+  - [Deploy **KubeStellar** in an **OpenShift** cluster](#deploy-kubestellar-in-an-openshift-cluster)
+
+## Deploy **KubeStellar** in a **Kubernetes** cluster (**Kind** cluster)
+
+[Create a **Kind** cluster with the `extraPortMappings` for port `1024` and an **nginx** ingreass with SSL passthrough.](../yaml/README.md)
+
+Deploy **KubeStellar** `stable` in a `kubestellar` namespace:
+
+```shell
+helm install kubestellar . --values values-kubernets.yaml

```

Comment:
Those two value files differ only in one string.  The way this is usually handled is to include a set of default values in the chart (see https://github.com/kcp-dev/helm-charts/blob/main/charts/kcp/values.yaml. for an example) and then tell users when and why as well as how to override defaults.  https://helm.sh/docs/helm/helm_install/ documents how to invoke `helm install` and provide overrides.  For example,

```shell
helm install kubestellar . --set-string clusterType=Kubernetes
```

Reply:
ok, I'll change the scripts and documentation accordingly